src/dispatch/plugins/dispatch_slack/case/messages.py

```diff
@@ -41,12 +41,22 @@
 
 
 def create_case_message(case: Case, channel_id: str) -> list[Block]:
+    # TODO we should probably restrict the possible colors to make this work
```

Comment:
Or the names of the priorities and map those to colors.

Reply:
The names are user-defined. So they could be anything :-/.